src/style.css

```diff
@@ -1 +1,96 @@
-/* add styles here */
+* {
+  font-weight: 300;
```

Comment:
Can we just replace it with a font-family: Avenir? Because you already set a default font-weight in a font-face below